packages/core/src/fhir-to-cda/cda-templates/components/__tests__/xmls/encounters-section-single-entry.xml

```diff
@@ -65,6 +69,13 @@
 			<assignedEntity>
 				<id root="placeholder-ORG-OID" extension="${practitionerId}">
 				</id>
+				<code code="207Q00000X" codeSystem="2.16.840.1.113883.6.101" displayName="FAMILY MEDICINE PHYSICIAN">
+					<originalText>
+						FAMILY MEDICINE PHYSICIAN
+					</originalText>
+					<translation code="207Q00000X" codeSystem="2.16.840.1.113883.6.101" displayName="FAMILY MEDICINE PHYSICIAN">
+					</translation>
```

Comment:
why do we have a translation that is identical to the original code? Seems like something funny in the logic since there is only one code in the coding array.

Reply:
Fixed